packages/markdown/processor/parse/variable-parser.js

```diff
@@ -1,4 +1,4 @@
-const { VARIABLE_REGEXP } = require('@readme/variable');
+const VARIABLE_REGEXP = /(?:\\)?<<([-\w:\s]+)(?:\\)?>>/.source;
```

Comment:
How come you made this change?

Reply:
not a good reason, I can tell you that... 😬 will revert.